src/test/java/service/impl/FruitOperationPerformerTest.java

```diff
@@ -0,0 +1,86 @@
+package service.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import db.FruitStorage;
+import db.FruitStorageDao;
+import db.StorageDao;
+import java.util.List;
+import java.util.Map;
+import model.FruitTransaction;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import service.Performer;
+import strategy.FruitTransactionStrategy;
+import strategy.TransactionStrategy;
+import strategy.operation.FruitBalanceOperation;
+import strategy.operation.FruitPurchaseOperation;
+import strategy.operation.FruitReturnOperation;
+import strategy.operation.FruitSupplyOperation;
+import strategy.operation.OperationHandler;
+
+class FruitOperationPerformerTest {
+    private static FruitStorage FRUIT_STORAGE;
+    private static StorageDao<String,Integer> STORAGE_HANDLER;
+    private static Map<FruitTransaction.Operation,
+            OperationHandler<String, Integer>> OPERATION_HANDLER_MAP;
+    private static TransactionStrategy MAKE_TRANSACTION;
```

Comment:
They are not declared as final; why are they named in uppercase as if they were constants? According to the Google Java Style Guide, static final members should only be in uppercase if they are constants, which implies deep immutability